src/vivarium_nih_us_cvd/components/risk_correlation.py

```diff
@@ -142,3 +107,49 @@ def update_correlation_data(self, correlation_data: pd.DataFrame) -> pd.DataFram
             correlation_data[new_column] = correlation_data[original_column].values
 
         return correlation_data
+
+
+class JointPAF(Component):

```

Comment:
No changes to the above deleted code from RiskCorrelation - I simply moved the stuff here

Reply:
I think a cleaner solution would have been to create a component `PAFCalculationRiskCorrelation(RiskCorrelation)` that overrides these two method with `pass` and including that in the paf calculation sim instead of the normal one.

This is fine though.